File: RADWIMPS.py
#!/usr/bin/env python3


class RADWIMPS:
    @classmethod
    def then(cls):
        print('前', end='')
        return cls

    @classmethod
    def 世(cls):
        print('世')
        return cls


async def wait_while_they_play(future):
    await RADWIMPS.then().then().then().世()
    future.set_result(True)


async def main():
    loop = asyncio.get_running_loop()
    future = loop.create_future()
    loop.create_task(wait_while_they_play(future))
    await future


if __name__ == '__main__':
    asyncio.run(main())

File: RADWIMPS.zig
const std = @import("std");

pub fn main() void {
    const RADWIMPS = struct {
        pub fn then(self: @This()) @This() {
            std.debug.print("前", .{});
            return self;
        }

        pub fn 世(self: @This()) @This() {
            std.debug.print("世\n", .{});
            return self;
        }
    };

    var radwimps = RADWIMPS{};
    radwimps = radwimps.then().then().then().世();
}
